File: nelio-alves-algoritmo-e-logica-de-programacao/cpp/exercicios/media_pares.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    int N, soma, cont;
    double media;

    cout << "Quantos elementos vai ter o vetor? ";
    cin >> N;

    int vet[N];

    for (int i = 0; i < N; i++) {
        cout << "Digite um numero: ";
        cin >> vet[i];
    }

    cont = 0;
    soma = 0;
    for (int i = 0; i < N; i++) {
        if (vet[i] % 2 == 0) {
            soma = soma + vet[i];
            cont++;
        }
    }

    if (cont == 0) {
        cout << "NENHUM NUMERO PAR" << endl;
    } else {
        media = (double) soma / cont;

        cout << fixed << setprecision(1);
        cout << "MEDIA DOS PARES = " << media << endl;
    }

    return 0;
}

File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/media_pares.py
N: int; soma: int; cont: int
media: float

N = int(input("Quantos elementos vai ter o vetor? "))

vet: [int] = [0 for x in range(N)]

for i in range(N):
	vet[i] = int(input("Digite um numero: "))

cont = 0
soma = 0
for i in range(N):
	if vet[i] % 2 == 0:
		soma = soma + vet[i]
		cont = cont + 1

if cont == 0:
	print("NENHUM NUMERO PAR")
else:
	media = float(soma) / cont

	print(f"MEDIA DOS PARES = {media:.1f}")